main.css

```diff
@@ -0,0 +1,293 @@
+	vertical-align: middle;
+}
+.li_bubble:before {
+	content: url(/singolo/images/services-item-5--nocircle.svg);
+	vertical-align: middle;
+}
+.li_phone:before {
+	content: url(/singolo/images/services-item-4--nocircle.svg);
+	vertical-align: middle;
+}
+#services ul li h4 {
+	float: right;
+	width: 220px;
+	margin: 0px;
+}
+#services ul li p {
+	float: right;
+	width: 220px;
+	text-align: justify;
+}
+#portfolio {
+	background: #2d303a;
+	overflow: hidden;
+	border-bottom: 6px solid #323746;
+}
+#portfolio .inner {
+	padding-bottom: 52px;
+}
+#portfolio .inner h3 {
+	margin-bottom: 12px;
+}
+#portfolio ul.portfolio-menu {
+	width: 100%;
+	overflow: hidden;
+}
+#portfolio ul.portfolio-menu li {
+	float: left;
+	display: inline;
+	margin: 0 10px 0 0;
+	font-size: 12px;
+	font-weight: 300;
+}
+#portfolio ul.portfolio-menu li a {
+	border-radius: 5px;
+	border: 1px solid #666d89;
+	color: #767e9e;
+	padding: 4px 7px;
+	height: 100%;
+	display: block;
+}
+#portfolio ul.portfolio-menu li a:hover,
+#portfolio ul.portfolio-menu li.current a {
+	border-radius: 5px;
+	border: 1px solid #c5c5c5;
+	color: #dedede;
+	transition-duration: 0.5s;
+}
+#portfolio ul.portfolio {
+	width: 100%;
+	padding: 19px 0 0 0;
+}
+#portfolio ul.portfolio li {
+	float: left;
+	display: inline;
+	margin: 0 15px 15px 0;
+}
+#portfolio ul.portfolio li a {
+	height: 100%;
+	display: block;
+}
+#portfolio ul.portfolio li a:hover {
+	opacity: 0.6;
+	filter: alpha(opacity=60);
+	transition-duration: 0.5s;
+}
+#footer {
+	background: #2d303a;
+	overflow: hidden;
+}
+#footer .inner {
+	padding-top: 22px;
+	padding-bottom: 22px;
+}	
+#footer p {
+	color: #666d89;
+}	
+#footer p a {
+	line-height: 32px;
+	color: #666d89;
+}	
+#footer p a:hover {
+	color: #f06c64;
+	transition-duration: 0.5s;
+}	
+#footer ul {
+	float: right;
+}	
+#footer ul li span {
+	font-family: "fontello";
+	display: inline-block;
+	width: 30px;
+	height: 30px;
+	line-height: 30px;
+	margin: 0 10px 0 0;
+	background: transparent;
+	border-radius: 50%;
+	border: 1px solid #6a718c;
+	color: #6a718c;
+	text-align: center;
+	font-size: 16px;
+	cursor: pointer;
+}
+#footer ul li span:hover {
+	border: 1px solid #f06c64;
+	color: #f06c64;
+	transition-duration: 0.5s;	
+}
+.icon-twitter:before { content: url(/singolo/images/ico-twitter--nocircle.png); vertical-align: middle; }
+.icon-facebook:before { content: url(/singolo/images/ico-facebook--nocircle.png); vertical-align: middle; }
+.icon-gplus:before { content: url(/singolo/images/ico-google-plus--nocircle.png); vertical-align: middle; }
+.icon-linkedin:before { content: url(/singolo/images/ico-linkedin--nocircle.png); vertical-align: middle; }
+@media (max-width: 375px) {
+.icon {
+	display: block;
+}
+.header {
+	justify-content: flex-start;
+	align-items: flex-start;
+}
+.header .nav-menu {
+	display: none;
+}
+.header.responsive {
+flex-direction: column;
+
+}
+.header.responsive .nav-menu {
+	display: flex;
+	flex-direction: column;
+	
+}
```

Comment:
It seems that class .responsive is never added, so it can be removed.

Reply:
fixed

Reply:
this class is needed for hamburger menu